packages/mdast/mail-templates/src/newsletter/schema.jsx

```diff
@@ -371,31 +260,22 @@ const createNewsletterSchema = ({
                     start: node.start,
                   },
                 }),
-                editorModule: 'list',
                 rules: [
                   {
                     matchMdast: matchType('listItem'),
                     component: ListItem,
-                    editorModule: 'listItem',
                     rules: [listParagraph],
                   },
                 ],
               },
               {
                 matchMdast: matchType('thematicBreak'),
                 component: HR,
-                editorModule: 'line',
-                editorOptions: {
-                  insertButtonText: 'Trennlinie',
-                  insertTypes: ['PARAGRAPH'],
-                },
-                isVoid: true,
               },
             ],
           },
           {
             matchMdast: () => false,
-            editorModule: 'specialchars',
```

Comment:
What are all these deletions? Are they not needed anymore?

Reply:
ah, the mail templates actually don't need the editorModule/editorOptions since we only use them to render emails, not to work in Publikator. (We use the templates from styleguide, and those since have the editorModule/editorOptions).
So I deleted them. Should do this for all the mdast/mail-templates.

Reply:
Nice!